### PR TITLE
UNFCCC & EPPO source update

### PR DESCRIPTION
**Pull request by maryarbab**
Is this a new addition or an update to an existing factor?

Update

What category or categories does it affect?

Electricity 

What?

Updates to the year, source link, value, and description fields.

Why?

To link to the most recent UNFCCC and EPPO sources.

### CODE REVIEW

**Review by greenercode — DISMISSED** [review]

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -437,8 +437,8 @@ Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.437
...
-Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.445,,EPPO,2019,TH,2021/07/30,Emission intensity of grid mix as reported. Published by the Energy Policy and Planning Office (EPPO) of Thailand in 2019. Includes production fuel mix factor. Includes generation factor.,http://www.eppo.go.th/index.php/en/,electricity_generation
-Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.004,,UNFCCC,2018,CA-PE,2021/07/30,Emission intensity of grid mix as reported. Published in the UN Framework Convention on Climate Change 2020. Includes generation factor.,https://data.ec.gc.ca/data/substances/monitor/canada-s-official-greenhouse-gas-inventory/Emission_Factors.pdf,electricity_generation
+Energy,Electricity,electricity-energy_source_grid_mix,Grid mix,kg-CO2e/kWh,0.442,,EPPO,2020,TH,2021/07/30,Emission intensity of grid mix as reported. Published by the Energy Policy and Planning Office (EPPO) of Thailand. Available in Table 9.1-15 of the source link. Includes generation factor.,http://www.eppo.go.th/index.php/en/en-energystatistics/co2-statistic,electricity_generation
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:440]: Love this level of detail! We are getting people respond really well to our transparency and consistency - this is great.

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -1095,8 +1095,8 @@ Materials,Mined Materials,mined_materials-type_copper_nickel_lead_zinc,Copper/ni
...
-Materials,Mining ,mining-type_well_drilling,Well drilling,kg-CO2e/USD,0.543,,EPA,2020,US,2021/10/12,Emission intensity of supply chain (with margins i.e. cradle to shelf) in US dollars spend on: well drilling. This factor is representative of the described commodity (equivalent to a goods or services category) and was calculated from 2016 data applying IPCC 4th Assessment Report conversion factors.,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
+Materials,Mining,mining-type_other_support_activities_for_mining,Other support activities for mining,kg-CO2e/USD,0.613,,EPA,2020,US,2021/10/12,Emission intensity of supply chain (with margins i.e. cradle to shelf) in US dollars spend on: other support activities for mining. This factor is representative of the described commodity (equivalent to a goods or services category) and was calculated from 2016 data applying IPCC 4th Assessment Report conversion factors.,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1098]: this should be removed to avoid conflicts

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -1095,8 +1095,8 @@ Materials,Mined Materials,mined_materials-type_copper_nickel_lead_zinc,Copper/ni
...
+Materials,Mining,mining-type_well_drilling,Well drilling,kg-CO2e/USD,0.543,,EPA,2020,US,2021/10/12,Emission intensity of supply chain (with margins i.e. cradle to shelf) in US dollars spend on: well drilling. This factor is representative of the described commodity (equivalent to a goods or services category) and was calculated from 2016 data applying IPCC 4th Assessment Report conversion factors.,https://edg.epa.gov/metadata/catalog/search/resource/details.page?uuid=https://doi.org/10.23719/1517796,cradle_to_shelf
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1099]: this should be removed to avoid conflicts

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -2093,4 +2093,4 @@ Waste,Plastic Waste,waste_type_pvc_including_forming-disposal_method_open_loop,P
...
-Water,Water Treatment,water-treatment,Water treatment,kg-CO2e/m3,0.272,,BEIS,2021,GB,2021/07/23,Emission Intensity of water treatment in the UK in 2021. Based on emissions associated with offices/transport/treatment of wastewater. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,upstream
+Water,Water Treatment,water-treatment,Water treatment,kg-CO2e/m3,0.272,,BEIS,2021,GB,2021/07/23,Emission Intensity of water treatment in the UK in 2021. Based on emissions associated with offices/transport/treatment of wastewater. Available in BEIS Greenhouse Gas Reporting: Conversion Factors 2021.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021,upstream
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:2096]: this should be removed to avoid conflicts

**Review by patstirling — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 2 above)
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv]: Done done;)

**Review by AnastasiaLobanova — APPROVED** [review]

**Review by patstirling — APPROVED** [review]

**Review by patstirling — APPROVED** [review]

**Pull request merged**